File: Modul 2/soal_5/PRAK205-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
import math

banyak_variable = 2
inputs = []

while len(inputs) < banyak_variable:
    line = input()

    new_line = line.split()

    for value in new_line:
        inputs.append(float(value))

sisi_a, sisi_b = inputs
pythagoras = (sisi_b**2)-(sisi_a**2)
sisi_c = math.sqrt(pythagoras)
keliling = sisi_a+sisi_c+sisi_b
tinggi = sisi_a
alas = sisi_c
luas = 0.5*alas*tinggi

print("Alas = %d cm\nTinggi = %d cm\nKeliling = %d cm\nLuas = %d cm^2\n"%(alas, tinggi, keliling, luas))

File: Modul 2/soal_5/PRAK205-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>
#include <math.h>
int main(){
    int sisi_a = 0;
    int sisi_b = 0;
    scanf("%d %d", &sisi_a, &sisi_b);
    float pythagoras = (sisi_b*sisi_b)-(sisi_a*sisi_a);
    float sisi_c = sqrt(pythagoras);
    int keliling = sisi_a+sisi_c+sisi_b;

    int tinggi = sisi_a;
    int alas = sisi_c;
    int luas = 0.5*alas*tinggi;
    printf("Alas = %d cm\nTinggi = %d cm\nKeliling = %d cm\nLuas = %d cm^2\n", alas, tinggi, keliling, luas);
}